.infrastructure/12_certs.tf

```diff
@@ -0,0 +1,14 @@
+module "acm" {
```

Comment:
Is this certificate going to be linked to a CDN?
Because in that case, [check this documentation out](https://github.com/terraform-aws-modules/terraform-aws-acm?tab=readme-ov-file#usage-with-cloudfront)

Reply:
no, the certificate is for linked to ALB. CMS not have a CDN